pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -532,6 +544,15 @@ def _stop_AWG(self, AWG):
         else:
             raise ValueError('Unsupported AWG type: {}'.format(type(obj)))
 
+    def _is_AWG_running(self, AWG):
+        obj = self.AWG_obj(AWG=AWG)
+        if isinstance(obj, Tektronix_AWG5014):
+            return obj.get_state() != 'Idle'
+        elif isinstance(obj, UHFQC):
```

Comment:
If I understand correctly this will directly break the use of the Pulsar for use with any UHFQC. As far as I know you guys are the only ones using that but it might be worth checking that.

Reply:
Yes, it is true that this will break support for the UHFQC as a slave AWG. But since we are not using it currently, I prefer to leave it unimplemented rather than implement something I could not immediately test.